### DISCUSSION TITLE
4 x 72 Led with ESP8266

### DISCUSSION THREAD

**Discussion by trbLeeciN** [Makuna/NeoPixelBus#652] [Q&A] [answered]
Hello,

I have been trying to solve my problem almost 3 weeks. 
I tried fastled, neopixel and finally I am here :)
I am using nodemcu exp8266 in my project. I am getting 4 data from internet via Wifi every 1 seconds.
I need to show these data in 7segment display seperately. Each 7 segment display includes 72 leds, in total I need to 288 leds to drive.
At first I was aiming to use 4 different pins to drive 4 different display, however because of Wifi issues, I think I had to use GPIO3. Otherwise it crashes( The reason I couldn't use Neopixel)
So my question is, Can I drive 288 leds from 1 gpio in every 1 seconds and mean time get data from internet.
Or are there any other solutions to drive them 4 seperate gpios?

Thanks for your helps

**Comment by Makuna** [comment, marked as answer]
[FAQ #5: How fast can I update my NeoPixels](https://github.com/Makuna/NeoPixelBus/wiki/FAQ-%235)
ESP8266 by default unless you choose specific hardware names will use the DMA hardware. (the FAQ references this as NeoEsp8266Dma800KbpsMethod).  The key thing is that CPU isn't used much on ESP8266 by default and the FAQ describes this.  So, it's more about how much time you have to wait on the bus to transmit the data to the pixels.
```
   RGB microseconds = pixelcount x 30 + 300 + 100
   RGBW microseconds = pixelcount x 40 + 300 + 100
```

**Reply by trbLeeciN** [reply]
Thank you for quick reply, you the best.
